### PR TITLE
It's main.js not bundle.js

### PR DESCRIPTION
**Pull request by bangzek**

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
This needs testing on PR

**Comment by bangzek** [comment]
Or you could add the diff from main.js to bundle.js in the getting-started.md

File: src/content/guides/output-management.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ __dist/index.html__
 +     <script src="./print.bundle.js"></script>
     </head>
     <body>
--     <script src="./bundle.js"></script>
+-     <script src="./main.js"></script>
```
Comment (EugeneHlushko) [inline src/content/guides/output-management.md]: This was correct, it extends asset-management guide and it says so at the top of the guide

File: src/content/guides/output-management.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ __webpack.config.js__
 +     print: './src/print.js'
 +   },
     output: {
--     filename: 'bundle.js',
+-     filename: 'main.js',
```
Comment (EugeneHlushko) [inline src/content/guides/output-management.md]: this was also correct, please see asset management guide

**Comment by EugeneHlushko** [comment]
screen shot 2018-11-05 at 6 18 16 pm

**Comment by EugeneHlushko** [comment]
Closing for now, please comment if i have missed anything

**Pull request closed without merging**